prompting/baseminer/miner.py

```diff
@@ -108,7 +108,11 @@ def __init__(self, config=None, axon=None, wallet=None, subtensor=None):
             bt.logging.info(f"Running miner on uid: {self.my_subnet_uid}")
 
         # The axon handles request processing, allowing validators to send this process requests.
-        self.axon = axon or bt.axon(wallet=self.wallet, port=self.config.axon.port)
+        self.axon = axon or bt.axon(
+            wallet=self.wallet,
+            port=self.config.axon.port,
```

Comment:
How is this value going to be defined considering that the config for `axon.port` would be removed in #82?

Reply:
In #82 we're removing a duplicate call to add_args. Because there is a try-catch around the add_args from the `subtensor` class, it doesn't raise the exception, but it also doesn't add any other axon arguments. 
The call to add the port argument/flag in this repo is interfering with the bittensor package adding the rest of the arguments.